sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/_models/_messages.py

```diff
@@ -0,0 +1,199 @@
+        type (str): The type of the content, which is always 'tool_call'.
+        name (str): The name of the tool.
+        tool_call_id (str): The ID of the tool call.
+        arguments (Optional[Dict[str, Any]]): The arguments for the tool call.
+    """
+    type: str = "tool_call"
+    name: str
+    tool_call_id: str
+    arguments: Optional[Dict[str, Any]] = None
+
+class ToolResultContent(AIContent):
+    """
+    Represents tool result content.
+
+    Attributes:
+        type (str): The type of the content, which is always 'tool_result'.
+        tool_call_id (str): The ID of the tool call.
+        results (Optional[Any]): The results of the tool call.
+    """
+    type: str = "tool_result"
+    tool_call_id: str
+    results: Optional[Any] = None
+
+class FileContent(AIContent):
+    """
+    Represents file content.
+
+    Attributes:
+        type (str): The type of the content, which is always 'file'.
+        file_name (Optional[str]): The name of the file.
+        mime_type (Optional[str]): The MIME type of the file.
+        uri (Optional[str]): The URI of the file.
+        data_uri (Optional[str]): The data URI of the file.
+        data (Optional[bytes]): The binary data of the file.
+    """
+    type: str = "file"
+    file_name: Optional[str] = None
+    mime_type: Optional[str] = None
+    uri: Optional[str] = None
+    data_uri: Optional[str] = None
+    data: Optional[bytes] = None
+
+class AudioContent(FileContent):
+    """
+    Represents audio content.
+
+    Attributes:
+        type (str): The type of the content, which is always 'audio'.
+        duration (Optional[short]): The duration of the audio content.
+    """
+    type: str = "audio"
+    duration: Optional[int] = None
+
+class ImageContent(FileContent):
+    """
+    Represents image content.
+
+    Attributes:
+        type (str): The type of the content, which is always 'image'.
+        width (Optional[int]): The width of the image.
+        height (Optional[int]): The height of the image.
+    """
+    type: str = "image"
+    width: Optional[int] = None
+    height: Optional[int] = None
+
+class VideoContent(FileContent):
+    """
+    Represents video content.
+
+    Attributes:
+        type (str): The type of the content, which is always 'video'.
+        duration (Optional[int]): The duration of the video.
+        width (Optional[int]): The width of the video.
+        height (Optional[int]): The height of the video.
+    """
+    type: str = "video"
+    duration: Optional[int] = None
+    width: Optional[int] = None
+    height: Optional[int] = None
+
+class ChatMessage(BaseModel):
+    """
+    Represents a chat message.
+
+    Attributes:
+        user_id (Optional[str]): The ID of the user who sent the message.
+        agent_id (Optional[str]): The ID of the agent who sent the message.
+        message_id (str): The ID of the message.
+        completion_id (Optional[str]): The ID of the completion associated with the message.
```

Comment:
From our conversation earlier, would `completion_id` somehow incorporate the `thread_id` into itself, resulting in us no longer needing the latter if former is given?